.changeset/forty-buses-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
```

Comment:
Maybe leave this empty, we don't necessarily have to release this since its not a feature?